pkg/reconciler/kubernetes/tektonchain/controller.go

```diff
@@ -80,12 +80,16 @@ func NewExtendedController(generator common.ExtensionGenerator) injection.Contro
 
 		logger.Info("Setting up event handlers for Tekton Chain")
 
-		tektonChaininformer.Get(ctx).Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
+		if _, err := tektonChaininformer.Get(ctx).Informer().AddEventHandler(controller.HandleAll(impl.Enqueue)); err != nil {
+			logging.FromContext(ctx).Panicf("Couldn't register TektonChain informer event handler: %w", err)
```

Comment:
@piyush-garg we can use the existing `logger`. I think `Fatalf` is enough. Do we need `Panicf`?

this is applicable to other files(changes) too.

Reply:
i will fix the logger.

Panicf will give stacktrace too, where it is breaking, because these will be there in multiple reconcilers

Reply:
I dont have a strong opinion, fatalf will also do the job.

Reply:
ok, I am fine with either
https://pkg.go.dev/go.uber.org/zap@v1.24.0/zapcore#Level
```
	// PanicLevel logs a message, then panics.
	PanicLevel (4)
	// FatalLevel logs a message, then calls os.Exit(1).
	FatalLevel (5)
```